attributes/default.rb

```diff
@@ -49,13 +30,10 @@
 when 'mac_os_x'
   default['java']['install_flavor'] = 'homebrew'
   default['java']['homebrew']['cask'] = 'java'
-else
-  default['java']['install_flavor'] = 'openjdk'
+  # else
+  #   Chef::Log.info('The default recipe is no longer recommended. Please use the install resources instead')
```

Comment:
?

Reply:
Just in the middle of trying to refactor these attributes out. I think this will go in the next push or two